src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversRobot = {};
```

Comment:
Незрозуміло, що таке invers
```suggestion
  const inversedRobot = {};
```